.changes/fix-macos-window-buttons.md

```diff
@@ -0,0 +1,6 @@
+---
+"tao": patch
+tag: "bug"
```

Comment:
```suggestion
"tao": patch
```

tags are really only used in the main tauri repo and the syntax is `patch:bug` for future reference